CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [1.7](https://github.com/arborx/arborx/tree/v1.7) (2024-08-30)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.6...v1.7)
+
+**New features:**
+- Add a distributed tree nearest query with a callback [EXPERIMENTAL] [\#1075](https://github.com/arborx/ArborX/pull/1075)
+- Add a distributed tree spatial query only taking a callback [\#733](https://github.com/arborx/ArborX/pull/733)
+
+**Build Changes:**
+- Require Kokkos 4.2.00 [\#1054](https://github.com/arborx/ArborX/pull/1054)
+
+**Enhancements:**
+- Implement box-triangle intersection [\#1059](https://github.com/arborx/ArborX/pull/1059)
+- Add tetrahedron geometry [EXPERIMENTAL] [\#1079](https://github.com/arborx/ArborX/pull/1079)
+- Expand KDOP to support different dimensions and precision [EXPERIMENTAL] [\#982](https://github.com/arborx/ArborX/pull/982)
+- Add 2D KDOP geometries [EXERIMENTAL] [\#1088](https://github.com/arborx/ArborX/pull/1088)
+- Improve performance of the distributed algorithms [\#1098](https://github.com/arborx/ArborX/pull/1098), [\#1103](https://github.com/arborx/ArborX/pull/1103)
```

Comment:
I find that quite vague

Reply:
Somewhat intentionally. The gist is that they should be faster, but there are a lot of details that are not user friendly.